Shell/TheoryFinder.cpp

```diff
@@ -164,10 +164,10 @@ class TheoryFinder::Backtrack
 public:
   /** code pointer */
   unsigned cp;
-  /** object on which the instruction should be executed */
-  const void* obj;
   /** position in the object stack */
   unsigned objPos;
+  /** object on which the instruction should be executed */
+  const void* obj;
 }; // TheoryFinder::Backtrack
```

Comment:
Aligment. The previous ordering of fields generates 8 bytes of padding, this ordering has no padding at all. Nothing seems to depend upon the precise order.